import/README.md

```diff
@@ -257,10 +258,10 @@ Returns: An infotable with one row, which is the most recent data entry.
 
 <b>updateTimeoutMinutes</b>: The maximum number of minutes the connector will run since the last data update before a connectivity error will be thrown.
 
-<b>useHyphensDataTables</b>: Changes how non-logged data is stored in Thingworx. If this is true, instead of dumping all non-logged data points into the same table, tags will be organized according to the placement of a hyphen.
+<b>useHyphens</b>: Changes how data is organised in Thingworx. If this is true, instead of dumping all data points into the Thing, tags will be organized according to the placement of a hyphen.
```

Comment:
```suggestion
<b>useHyphens</b>: Changes how data is organized in Thingworx. If this is true, instead of dumping all data points into the Thing, tags will be organized according to the placement of a hyphen.
```

Reply:
fixed